fe/fe-core/src/main/java/org/apache/doris/planner/BrokerScanNode.java

```diff
@@ -135,18 +144,26 @@ private static class ParamCreateContext {
 
     private List<ParamCreateContext> paramCreateContexts;
 
+    // For broker load and external broker table
     public BrokerScanNode(PlanNodeId id, TupleDescriptor destTupleDesc, String planNodeName,
                           List<List<TBrokerFileStatus>> fileStatusesList, int filesAdded) {
         super(id, destTupleDesc, planNodeName, NodeType.BROKER_SCAN_NODE);
         this.fileStatusesList = fileStatusesList;
         this.filesAdded = filesAdded;
+        if (ConnectContext.get() != null) {
```

Comment:
May be we don't need to get user info for query here.

Reply:
I think I known what you mean. Because for query, the actually tags we use is based on replica tag, not user level tag.
So may be we don't need user info for query.
But here I insist to add this assignment this, for unification purpose.